chromium_src/chrome/browser/prefs/pref_service_incognito_allowlist.cc

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/prefs/persistant_pref_names.h"
```

Comment:
I think we should list the individual pref includes here. Wrapping them all in persistent_pref_names.h just makes dependencies more confusing imo

Reply:
this is what upstream does and since this is chromium_src it gets more review and we can ensure that pref_names that it depends on are broken down into `:prefs` targets. We also don't want to add additional source files to sources.gni unless they are necessary which was recently discussed on slack

Reply:
discussed with @goodov and he also agrees that in this case we should drop persistant_pref_names.h